### ISSUE TITLE
Multiple tests, one failure

### ISSUE THREAD

**Issue by hajovonta** [emacsfodder/etd#1]
When running multiple tests for a function, one failed test prevents the other tests from running. It would be nice to have a summary of the failed/passed tests instead.

**Comment by jasonm23** [comment]
Show your example code too plese.

**Comment by jasonm23** [comment]
in this code...

```lisp
(def-example-group "Utility functions"
 (defexamples kurecolor-clamp
   (kurecolor-clamp 1 -1.0 1.0)    => 7         ;; show breakage here in test runner...?
   (kurecolor-clamp 2 -1.0 1.0)    => 1.0
   (kurecolor-clamp -2 -1.0 1.0)   => -1.0
   (kurecolor-clamp 0.1 -1.0 1.0)  => 0.1
   (kurecolor-clamp -0.1 -1.0 1.0) => -0.1))
```

Right?

Ok so, if each example was turned into individual tests, rather than one `ert-deftest` for a `defexamples` set the `ert` runner would break on the individual test.

I agree this would be preferable.

**Comment by hajovonta** [comment]
Yes, I had something like that in mind.

**Comment by jasonm23** [comment]
Ok, I've updated the live etd / examples on Kurecolor.

If they don't break I'll just port it back into here.

The API on that version has name prefixes for most things, (`etd-` public cmd/fn/var or `etd--` private/internal fn/var.)

The macros `defexamples` and `def-example-group`, remain though.

**Comment by jasonm23** [comment]
They broke... but that's optimism for you, not as good as test results.

**Comment by jasonm23** [comment]
Ok, I'm brimming with, not only optimism, **but also** test results now...

https://github.com/emacsfodder/kurecolor/runs/8266130278?check_suite_focus=true

**Comment by jasonm23** [comment]
Ok - tests generated per example line and #3 fixed in baab5b819fce02c9bf81ef28bdbc9347cfe1099a
